app/src/main/java/org/hackillinois/android/viewmodel/ProjectsViewModel.kt

```diff
@@ -0,0 +1,27 @@
+package org.hackillinois.android.viewmodel
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.ViewModel
+import org.hackillinois.android.database.entity.ProjectModel
+import org.hackillinois.android.repository.ProjectRepository
+
+class ProjectsViewModel : ViewModel() {
+    private val projectsRepository = ProjectRepository.instance
+
+    val dataScience = "Data Science"
+    val languages = "Languages"
+    val systems = "Systems"
+    val webDev = "Wev Development"
```

Comment:
**Web** Development

Reply:
Ooh nice catch, I was wondering why there wasn't anything loading under the Web Dev tab and just figured none of the projects in the API had that tag lol. Thanks!